modules/seqexec/server/src/main/scala/seqexec/server/gpi/GPI.scala

```diff
@@ -66,100 +66,81 @@ object GPI {
 
   private def gpiAoFlags(config: Config): Either[ExtractFailure, AOFlags] =
     for {
-      useAo      <- config.extract(INSTRUMENT_KEY / USE_AO_PROP).as[java.lang.Boolean]
-      useCal     <- config.extract(INSTRUMENT_KEY / USE_CAL_PROP).as[java.lang.Boolean]
+      useAo      <- config.extractAs[JBoolean](INSTRUMENT_KEY / USE_AO_PROP)
+      useCal     <- config.extractAs[JBoolean](INSTRUMENT_KEY / USE_CAL_PROP)
       aoOptimize <- config
-                      .extract(INSTRUMENT_KEY / AO_OPTIMIZE_PROP)
-                      .as[java.lang.Boolean]
+                      .extractAs[JBoolean](INSTRUMENT_KEY / AO_OPTIMIZE_PROP)
```

Comment:
Not a big deal but you could remove the leftover newlines in this method and in a few other places.

Reply:
ok, I've been running scalafmt to ensure the width is under 80, I'll check that again